implementations/rust/ockam/ockam_command/src/project/auth.rs

```diff
@@ -42,6 +42,21 @@ pub struct AuthCommand {
     trust_opts: TrustContextOpts,
 }
 
+fn parse_enroll_ticket(input: &str) -> Result<EnrollmentTicket> {
+    let ticket: EnrollmentTicket = match std::fs::read_to_string(input) {
+        Ok(s) => {
+            let decoded = hex::decode(s)?;
+            serde_json::from_slice(&decoded)?
+        }
+        Err(_) => {
+            let decoded = hex::decode(input)?;
+            serde_json::from_slice(&decoded)?
+        }
+    };
+
+    Ok(ticket)
```

Comment:
This could be simplified a little:

```rust
let decoded = match std::fs::read_to_string(input) {
    Ok(s) => hex::decode(s)?,
    Err(_) => hex::decode(input)?
};
Ok(serde_json::from_slice(&decoded)?)
```